hedera-node/src/main/java/com/hedera/services/context/SideEffectsTracker.java

```diff
@@ -72,6 +75,7 @@
 	private int numTouches = 0;
```

Comment:
We could rename this to `tokensTouchedSoFar`, and rename `touchedSoFar` below to `hbarBalanceTouchedSoFar`.

Reply:
Renamed!